### PR TITLE
Update for identify tool inside the vector chapter

### PR DESCRIPTION
**Pull request by selmaVH1**
The identify tool description for vector data. 
Discussed in [#8764](https://github.com/qgis/QGIS-Documentation/pull/8764).

### CODE REVIEW

File: docs/user_manual/working_with_vector/attribute_table.rst [hunk 1]
```diff
@@ -111,6 +111,23 @@ calculations to all or part of the features in the layer. This bar uses the same
 Calculator` (see :ref:`calculate_fields_values`).
 
 
+Using the Identify Tool with Attribute Table
+--------------------------------------------
+
+The |identify| :guilabel:`Identify features` tool can be used to display all attributes
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/attribute_table.rst:117]: here or next paragraph, can you ref the identify features section?

File: docs/user_manual/working_with_vector/attribute_table.rst [hunk 2]
```diff
@@ -111,6 +111,23 @@ calculations to all or part of the features in the layer. This bar uses the same
...
+of a feature from the attribute table in the :guilabel:`Identify Results` dialog.
+
+To use the :guilabel:`Identify features` tool for vector layers, follow these steps:
+
+* Select the vector layer in the Layers panel.
+* Click on the :guilabel:`Identify features` tool in the toolbar or press :kbd:`Ctrl+Shift+I`.
+* Click on a feature in the map view.
+
+:guilabel:`Identify Results` dialog will open and display information about
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/attribute_table.rst:126]: Let's use numbered list for steps
```suggestion
#. Select the vector layer in the Layers panel.
#. Click on the :guilabel:`Identify features` tool in the toolbar or press :kbd:`Ctrl+Shift+I`.
#. Click on a feature in the map view.
   The :guilabel:`Identify Results` dialog will open and display information about the clicked feature.
```
Then we need to provide what type of results is returned for vector layer

**Review by DelazJ — COMMENTED** [review]
Thanks @selmaVH1 for taking care of these adjustments.

IMHO, the main point in mentioning the identify features tool in each data provider chapter is to stop patching the "[feature information](https://docs.qgis.org/testing/en/docs/user_manual/introduction/general_tools.html#feature-information)" section with various "_if this format, you also have that_" and fully highlight specific outputs of each format in a dedicated place. Indeed, depending on whether you query a raster, vector or mesh you won't get the same type of contents for "attributes" and "derived" results, as well as the contextual menu. I am thus missing the vector outputs here i.e. most of the feature information section (and probably the text about contextual menu items). Or did I misunderstand the idea?

**Comment by selmaVH1** [comment]
> IMHO, the main point in mentioning the identify features tool in each data provider chapter is to stop patching the "[feature information](https://docs.qgis.org/testing/en/docs/user_manual/introduction/general_tools.html#feature-information)" section with various "_if this format, you also have that_" and fully highlight specific outputs of each format in a dedicated place. Indeed, depending on whether you query a raster, vector or mesh you won't get the same type of contents for "attributes" and "derived" results, as well as the contextual menu. I am thus missing the vector outputs here i.e. most of the feature information section (and probably the text about contextual menu items). Or did I misunderstand the idea?

Hi @DelazJ, thanks for the review. Your feedback clarifies what needs to be addressed, it seems I misunderstood the idea.

**Pull request merged**